projects/driver_controls/src/stm32f0xx/ads1015.c

```diff
@@ -0,0 +1,201 @@
+#include "ads1015.h"
```

Comment:
I would recommend adding a comment explaining the basic idea behind the implementation - rotating channels, alert pin, etc.

Reply:
Hmm, this seems to be more like usage instructions. I think this is valuable for explaining the overall use of the module, but it would also be nice to cover the hardware details that need to be handled - rotating channels, the alert pin, etc. Remember that the point of these comments is for people with no background on the chip to get a quick idea of what's important.

Reply:
I think this comment block got a little out of hand. :P

I think it'd be more useful for this section to explain the underlying concepts rather than what each function does. For example, I think they could figure out that communication is done through I2C based on the code.

Basically, I'm looking for something more along the lines of why things are implemented (reasoning) more than how they're implemented.